client/jetpack-cloud/components/layout/header.tsx

```diff
@@ -39,8 +39,36 @@ export default function LayoutHeader( { showStickyContent, children }: Props ) {
 
 	const outerDivProps = divRef ? { ref: divRef as React.RefObject< HTMLDivElement > } : {};
 
+	const [ minHeaderHeight, setMinHeaderHeight ] = useState( 0 );
+
+	// To avoid shifting the layout when displaying sticky content,  we will need to
+	// keep track of our Header height and set it as the minimum viewport height.
+	useEffect(
+		() => {
+			const headerRef = outerDivProps?.ref?.current;
```

Comment:
I think it's better to check if showStickyContent is true and then proceed.

Reply:
After attempting to check for `showStickyContent` before registering a callback, I found that the fix didn't work. My suspicion is that we failed to set the minimum height on time before the sticky header appeared causing the layout shift. I've decided to leave this implementation as it is since I didn't encounter any performance issues with it.

But thanks for suggesting this 👍